### PR TITLE
Add unit test for PASCALVOC Dataset Loader classes

### LINKED ISSUE #1142: Add unit test for PASCALVOC Dataset Loader classes
For do not have unit test for PASCALVOC dataset loader classes.
https://github.com/blue-oil/blueoil/tree/master/tests/unit/fixtures/datasets

`Pascalvoc2007`, `Pascalvoc2012`, `Pascalvoc20072012` classes need tests.

### PR DESCRIPTION
**Pull request by oatawa1**
## What this patch does to fix the issue.
Add test for PASCALVOC dataset loader classes.
- `Pascalvoc2007`
- `Pascalvoc2012`
- `Pascalvoc20072012`
- `PascalvocCustom`

Add dummy dataset for Pascalvoc2012.

## Link to any relevant issues or pull requests.
Closes #1142

### CODE REVIEW

**Comment by oatawa1** [comment]
@iizukak ping!

**Comment by iizukak** [comment]
Please use dummy image file for `2012_002012.jpg`.

**Comment by oatawa1** [comment]
@iizukak Thanks, I fixed it. Could you review again?

File: tests/unit/datasets_tests/test_pascalvoc.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
```
Comment (iizukak) [inline tests/unit/datasets_tests/test_pascalvoc.py:1]: We don't need this line.

File: tests/unit/datasets_tests/test_pascalvoc.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    assert dataset.available_subsets == ['train', 'validation', 'test']
+    assert dataset.files == [
+        'unit/fixtures/datasets/PASCALVOC_2007/VOCdevkit/VOC2007/JPEGImages/000085.jpg',
+        'unit/fixtures/datasets/PASCALVOC_2012/VOCdevkit/VOC2012/JPEGImages/2012_002012.jpg'
+    ]
+    assert dataset.annotations == [[[21, 69, 232, 257, 14], [337, 11, 162, 187, 14]],
+                                   [[190, 67, 145, 307, 14]]]
+
+
+def test_pascalvoccustom_object_detection():
```
Comment (iizukak) [inline tests/unit/datasets_tests/test_pascalvoc.py:59]: `test_pascalvoccustom_object_detection ` -> `test_pascalvoccustom` is not enough?

File: tests/unit/datasets_tests/test_pascalvoc.py (hunk 1 above)
Comment (oatawa1) [reply tests/unit/datasets_tests/test_pascalvoc.py]: Removed

File: tests/unit/datasets_tests/test_pascalvoc.py (hunk 2 above)
Comment (oatawa1) [reply tests/unit/datasets_tests/test_pascalvoc.py]: Thank you. 

`test_pascalvoccustom_object_detection` should be enough.

Fixed.

**Comment by oatawa1** [comment]
@iizukak Thank you.

**Comment by oatawa1** [comment]
@tfujiwar Thank you.

**Pull request merged**